index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <img src="/images/Welcome.png" alt="welcome button at the top left">
+        <img src="/images/Union.png" alt="a union of three vectors">
```

Comment:
- [ ] the hamburger button should be inside the `<nav>` tag cause its part of the navigation you should always use the semantic tags whenever possible

Reply:
thanks for that.
the hamburger really got me though